src/main/java/Bank.java

```diff
@@ -0,0 +1,28 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Bank {
+    static void creditBank(int credit) throws IOException {
```

Comment:
I would add credit time(maximum amount of payments). Now I can pay as many times as I want
![image](https://user-images.githubusercontent.com/39908109/102105138-b40ff480-3e37-11eb-8d83-3cacd44b1385.png)